aspnetcore/mvc/views/tag-helpers/builtin/resources/CacheTagHelper.md

```diff
@@ -0,0 +1,286 @@
+| Boolean             | "true"   	            |
+|                     | "false" (default) |
+
+The attribute `vary-by-user` specifies whether or not the cache should reset when the logged-in user (or Context Principal) changes. The current user is also known as the  Request Context Principal and can be viewed in a Razor view by referencing `@User.Identity.Name`.
+
+The example below looks at the current logged in user. When a different user is found, The content of the Cache Tag Helper is rendered again, and the cache is reset.  
+
+Usage Example:
+
+```html
+<Cache vary-by-user="true">
+    Current Time Inside Cache Tag Helper: @DateTime.Now
+</Cache>
+```
+
+> [!NOTE]
+>  Using the attribute `vary-by-user` maintains the contents in cache through a log-in and log-out cycle.  When using `vary-by-cookie` which references the `.AspNetCore.Identity.Application` as shown above, a log-in and log-out action invalidates the cache for the same authenticated user because a new cookie value is generated. If no user is authenticated, that state is considered a valid cache state, which means that no logged-in user is one cache state, and the contents will be maintained for that condition as well.
+
+- - -
+
+### vary-by
+
+| Attribute Type 	| Example Values            	|
+|----------------	|----------------             	|
+| String             | "@Model"  	            |
+
+
+The attribute `vary-by` allows for customization of what data gets cached. When the object referenced by the attribute's string value changes, the content of the Cache Tag Helper is updated. Often a string-concatenation of model values are assigned to this attribute.  Effectively, that means any update to any of the concatenated values invalidates the cache.
+
+The example below assumes the controller method rendering the view sums the integer value of the two route parameters, `myParam1` and `myParam2`, and returns that as the single model property. When this sum changes, the content of the Cache Tag Helper is rendered again, and the cache is reset.  
+
+Usage Example:
+
+
+*Controller*
+
+```csharp
+public IActionResult Index(string myParam1,string myParam2,string myParam3)
+{
+    int num1;
+    int num2;
+    int.TryParse(myParam1, out num1);
+    int.TryParse(myParam2, out num2);
+    return View(viewName, num1 + num2);
+}
+```
+
+*Index.cshtml*
+
+```html
+<Cache vary-by="@Model"">
+    Current Time Inside Cache Tag Helper: @DateTime.Now
+</Cache>
+```
+
+- - -
+
+### priority
+
+| Attribute Type 	| Example Values            	|
+|----------------	|----------------             	|
+| CacheItemPriority  | "High"   	            |
+|                    | "Low" |
+|                    | "NeverRemove" |
+|                    | "Normal" |
+
+The attribute `priority` provides cache ejection guidance to the built-in cache provider. If the web server experiences memory pressure and evicts one or more of your cached entities, it will evict Low priority cache entities first.
```

Comment:
I think if you go with the code version of "low" ... i.e., "Low" that you'll have to backtick it into code `Low`.